utils/streams/transfers.mjs

```diff
@@ -1,25 +1,51 @@
 import map from 'through2-map';
+import CombinedStream from 'combined-stream';
 import * as models from '../../models';
 
-export const strTransfer = (inStream, outStream, callback) => {
-  inStream.pipe(map(callback)).pipe(outStream);
+const defaultErrorHandler = err => {
+  if (err) {
+    console.log(err.message);
+  }
 };
 
-export const simpleTransfer = (inStream, outStream) => {
-  inStream.pipe(outStream);
+export const strTransfer = (inStream, outStream, callback, errorHandler = defaultErrorHandler) => {
+  inStream.pipe(map(callback)).pipe(outStream).on('error', errorHandler);
 };
 
-export const csvToJsonTransfer = (inStream, outStream) => {
-  const data = [];
+export const simpleTransfer = (inStream, outStream, errorHandler = defaultErrorHandler) => {
+  inStream.on('error', errorHandler).pipe(outStream);
+};
+
+export const csvToJsonTransfer = (inStream, outStream, errorHandler = defaultErrorHandler) => {
+  let buffer = '';
+  let model;
   inStream.on('data', chunk => {
-    data.push(chunk.toString());
-  }).on('end', () => {
-    const lines = data.join('').split('\r\n');
-    const Model = models[lines[0]];
-    const result = [];
-    for (let i = 1; i < lines.length; i++) {
-      result.push(Model.createCSV(lines[i]).toJSON());
+    const lines = chunk.toString().split('\r\n');
+    const data = [];
+    let i = 0;
+    let n = lines.length - 1;
+    let prefix = ',';
+    if (!buffer) {
+      model = models[lines[0]];
+      prefix = '[';
+      i++;
+    } else {
+      data[i] = buffer + data[i];

```

Comment:
Should be
`lines[i] = buffer + lines[i];`
The reason is that you operate over _lines_ first (line 41) and only after creating a model it is pushed to _data_ array. So _data_ keeps JSON while _lines_ keeps csv content instead